sdx-connector/src/main/java/com/sequenceiq/cloudbreak/saas/client/sdx/config/SdxSaasChannelConfig.java

```diff
@@ -0,0 +1,35 @@
+package com.sequenceiq.cloudbreak.saas.client.sdx.config;
+
+
+import static io.grpc.internal.GrpcUtil.DEFAULT_MAX_MESSAGE_SIZE;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.sequenceiq.cloudbreak.grpc.ManagedChannelWrapper;
+
+import io.grpc.ManagedChannelBuilder;
+
+@Configuration
+public class SdxSaasChannelConfig {
+
+    @Value("${saas.sdx.host:localhost}")
+    private String endpoint;
+
+    @Value("${saas.sdx.port:8982}")
```

Comment:
isn't there a port conflict?

Reply:
this is for the mock which is the same. there is no counterpart currently, since i am not aware of any deployment yet regarding saas. so configuration probably will happen later for this one.
but work still in prpgress